src/test/java/com/google/javascript/cl2dts/types.js

```diff
@@ -59,3 +59,13 @@ types.j = {a: 'a'};
  * @template K,V
  */
 types.objectWithGenericKeyType = function(obj) { };
+
+/**
+ * @const
```

Comment:
Well, then I don't understand this very well. I tried `@export` and that _doesn't_ appear in `compiler.getTopScope().getAllSymbols()` but I don't know why.
For now, I've added the comment explaining why `@const` is here.